lib/readline.js

```diff
@@ -380,7 +381,7 @@ Interface.prototype._tabComplete = function() {
     self.resume();
 
     if (err) {
-      // XXX Log it somewhere?
+      debug(err);
```

Comment:
Congratulations your 1st contribution. 

You would be better to print more information. e.g. 
```javascript
debug('tab complete error ', err);
```

like https://github.com/nodejs/node/blob/master/lib/repl.js#L230